serverless/eventing/tuning/override-config.adoc

```diff
@@ -13,6 +13,15 @@ You can override the default configurations for some specific deployments by mod
 The `replicas` spec cannot override the number of replicas for deployments that use the Horizontal Pod Autoscaler (HPA), and does not work for the `eventing-webhook` deployment.
 ====
 
+[NOTE]
+====
+You can only override probes that are defined in the deployment by default.
+
+All Serving deployments define a readiness and a liveness probe by default, with these exceptions:
```

Comment:
Should "Serving" be "Knative Serving"? If "serving" is just a general term, then I would suggest no caps.